### PR TITLE
PS-3803: Crash on AIO failure caused by uninitialized data members

### PR DESCRIPTION
**Pull request by dutow**
The count and pending data members weren't always initialized,
which caused crashes when the AIO object was destructed after a
failure.

### CODE REVIEW

File: storage/innobase/os/os0file.cc [hunk 1]
```diff
@@ -6571,6 +6575,8 @@ AIO::AIO(
 # ifdef LINUX_NATIVE_AIO
 	,m_aio_ctx(),
 	m_events(m_slots.size())
+	,m_pending(0)
```
Comment (percona-ysorokin) [inline storage/innobase/os/os0file.cc]: `,m_pending(NULL)`
`,m_count(NULL)`

**Review by percona-ysorokin — APPROVED** [review]
LGTM with minor stylistic comment

**Comment by percona-ysorokin** [comment]
Ready to be merged.

**Comment by dutow** [comment]
Jenkins: https://jenkins.percona.com/view/5.7/job/mysql-5.7-param/1801/

**Pull request merged**